src/utils/db_connector.py

```diff
@@ -63,7 +69,9 @@ def prep_import(self):
             soc = sys_info["name"][0]
             name = sys_info["workload_name"][0]
         else:
-            error("[database] Unable to parse SoC and/or workload name from sysinfo.csv")
+            console_error(
+                "[database] Unable to parse SoC and/or workload name from sysinfo.csv"
+            )
```

Comment:
Change to reflect new parameters in `console_error()`
```suggestion
            console_error(
                "database", "Unable to parse SoC and/or workload name from sysinfo.csv"
            )
```